lib/recurly/adjustment.php

```diff
@@ -25,6 +25,7 @@
  * @property mixed[] $tax_details The nested address information of the adjustment: name, type, tax_rate, tax_in_cents.
  * @property string $tax_code Optional field for EU VAT merchants and Avalara AvaTax Pro merchants. If you are using Recurly's EU VAT feature, you can use values of unknown, physical, or digital. If you have your own AvaTax account configured, you can use Avalara tax codes to assign custom tax rules.
  * @property string $product_code Merchant defined product code
+ * @property string $credit_reason_code Can be set on POST if adjustment is a credit (unit_amount_in_cents < 0). Allowed values: [general, service, promotional, refund, gift_card, write_off]. Defaults to "general".
```

Comment:
Nit but: Should we mention POST here? Seems like this should refer to create? Although I think all we allow is create so maybe just "Can be set if adjustment is a credit..."